### PR TITLE
Update office-365-ti.md

### PR DESCRIPTION
**Pull request by msbemba**
updated Office 365 Threat Intelligence connection

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5866

File: microsoft-365/security/office-365-security/office-365-ti.md [hunk 1]
```diff
@@ -58,6 +58,17 @@ Use [Explorer (and real-time detections)](threat-explorer.md) to analyze threats
 
 To view and use this report, in the Microsoft 365 Defender portal, go to **Email & collaboration** > **Explorer**.
 
+## Office 365 Threat Intelligence connection
+
+This feature is only available if you've an active Office 365 E5 or the Threat Intelligence add-on. For more information, see the Office 365 Enterprise E5 product page.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/office-365-ti.md]:
```suggestion
This feature is only available if you have an active Office 365 E5 subscription or the Threat Intelligence add-on. For more information, see the Office 365 Enterprise E5 product page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba Please copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Please help us here. Thanks

**Comment by tiburd** [comment]
2/15: sent email to author

**Review by MSFTTracyP — APPROVED** [review]
Thanks

**Pull request merged**